elyra/api/elyra.yaml

```diff
@@ -1,152 +1,232 @@
-swagger: '2.0'
+openapi: 3.0.1
 info:
   title: Elyra API
   description: The API for the Elyra extensions
-  version: '1'
   contact:
     name: Elyra
-    url: 'https://github.com/elyra-ai/elyra'
-produces:
-  - application/json
-consumes:
-  - application/json
-parameters:
-  namespace:
-    name: namespace
-    required: true
-    in: path
-    description: The name of the namespace
-    type: string
-  resource:
-    name: resource
-    required: true
-    in: path
-    description: The name of the resource in a given namespace
-    type: string
+    url: https://github.com/elyra-ai/elyra
+  version: "1"
+
 paths:
+
   /api/elyra.yaml:
     get:
-      produces:
-        - text/x-yaml
-      summary: Get API information (yaml)
       tags:
-        - api
+      - api
+      summary: Get API information (yaml)
       responses:
-        '200':
+        200:
           description: Returns a swagger specification in yaml
-  '/api/metadata/{namespace}':
-    parameters:
-      - $ref: '#/parameters/namespace'
+          content:
+            text/x-yaml:
+              schema:
+                type: string
+        5XX:
+          description: Unexpected error.
+
+  /api/namespace:
     get:
-      summary: Get metadata for a given namespace
       tags:
-        - metadata
+      - namespace
+      summary: Get current namespaces
       responses:
-        '200':
-          description: Returns the metadata instances for a given namespace
-          schema:
-            type: object
-            properties:
-              namespace:
-                type: array
-                description: The name of the namespace.
-                items:
-                  $ref: '#/definitions/MetadataResource'
-        '404':
+        200:
+          description: Returns the list of current namespaces
+          content:
+            application/json:
+              schema:
```

Comment:
i believe this path (`/api/namespace`) is missing `items`.
similar to how you have it defined for `/api/schema/{namespace}/{resource}` or `/api/metadata/{namespace}`. it would be something like:
```
items:
  $ref: '#/components/schemas/NamespaceResource'
```
where `NamespaceResource` would define the schema of what is returned by the endpoint.